.github/workflows/test_rust.yml

```diff
@@ -94,3 +91,23 @@ jobs:
         with:
           name: swf_images
           path: tests/**/*.png.updated
+
+  check-required:
+    needs: changes
+    if: needs.changes.outputs.src == 'false'
+    name: Test Rust ${{ matrix.rust_version }} / ${{ matrix.os }}
+    runs-on: ${{ matrix.os }}
+    continue-on-error: ${{ matrix.rust_version == 'nightly' || matrix.rust_version == 'beta' }}
+    strategy:
+      fail-fast: false
+      matrix:
+        rust_version: [stable]
+        os: [ubuntu-latest, windows-latest, macOS-latest]
+        include:
+          - rust_version: nightly
+            os: ubuntu-latest
+          - rust_version: beta
+            os: ubuntu-latest
+
+    steps:
+      - uses: actions/checkout@v2
```

Comment:
I think even `actions/checkout@v2` is redundant, and this job can be completely a no-op.
Maybe try an empty step list?
```yml
steps: []
```

Reply:
Not valid, sadly. https://github.com/midgleyc/ruffle/actions/runs/1254269776

> No steps defined in `steps` for the following jobs: check-required

Reply:
Gah, maybe a no-op shell command?
```yml
steps:
  - run: echo noop
```